core/src/main/java/com/linecorp/armeria/server/logging/AccessLogWriter.java

```diff
@@ -50,7 +56,7 @@ static AccessLogWriter combined() {
      * Returns disabled access log writer.
      */
     static AccessLogWriter disabled() {
-        return requestLog -> { /* No operation. */ };
```

Comment:
I belive we can revert this change because Java returns a singleton instance for a lambda that does not capture a variable outside. 
In addition, conventionally, we don't add a field in an `interface` to be free from breaking changes.

Reply:
A test shows me otherwise

```java
    @Test
    void testAccessLogger() {
        final AccessLogWriter accessLogWriter1 = log -> {
        };

        final AccessLogWriter accessLogWriter2 = log -> {
        };

        assert(accessLogWriter1 == accessLogWriter2);   <--- false
    }
```

@ikhoon what is your recommendation? creating a public DisbaledAccessLogWriter class?

Reply:
`accessLogWriter1` and `accessLogWriter2` are different objects.
I meant `AccessLogWriter.disabled()` always returns the same instance.
```java
assert AccessLogWriter.disabled() == AccessLogWriter.disabled();
```

Reply:
thanks for the clarification, updated!